### DISCUSSION TITLE
[Feature] Notes per face

### DISCUSSION THREAD

**Discussion by JesseWebDotCom** [immich-app/immich#4872] [Feature Request]
### The feature

I'd love to be able to add a note per face: ex. Jack's First Grade Teacher

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by Nickxsch** [comment]
This would also solve my 'problem' of wanting only the first name visible when viewing photos, but also seeing the last name when viewing a person (since there's no separate first and last name fields).

**Comment by Yecgaa1** [comment]
Photos hold my memories, but often when I take a photo with someone I don't know very well, I can't remember the story of the face even when I see the photo and the face. Therefore, I would like to allow users to write more information about the face, not only its name or nickname, but also the story of how I met it and its contact information, so that I can better carry my memories with it.
